libcodechecker/server/client_db_access_handler.py

```diff
@@ -50,7 +59,7 @@ def conv(text):
     return text.replace('*', '%')
 
 
-def process_report_filter_v2(report_filter):
+def process_report_filter_v2(report_filter, count_filter=None):
```

Comment:
I would add a comment here how the `count_filter` modifies the filter expression.